src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private static final String NOT_GET_THE_ACTOR_BY_ID = "Unable to get the actor by ID";
```

Comment:
Please be explicit in terms of constant names. Specify that this is a string to use in error message. (fix for all such constant names)